docs/install/kubernetes.md

```diff
@@ -13,7 +13,7 @@ To install Tracee with [Postee](https://github.com/aquasecurity/postee), simply
 ``` bash
 kubectl create \
   -f https://raw.githubusercontent.com/aquasecurity/postee/main/deploy/kubernetes/postee.yaml \
-  -f https://raw.githubusercontent.com/aquasecurity/tracee/{{ git.tag }}/deploy/kubernetes/tracee-postee/tracee.yaml
+  -f https://raw.githubusercontent.com/aquasecurity/tracee/main/deploy/kubernetes/tracee-postee/tracee.yaml
```

Comment:
If for some reason we change path to tracee.yaml the links will be broken. That's why we use {{ git.tag }} macros

Reply:
using the latest public documentation, in 6.5, the link is 404 not found.

`https://raw.githubusercontent.com/aquasecurity/tracee/tracee/deploy/kubernetes/tracee-postee/tracee.yaml`

https://aquasecurity.github.io/tracee/v0.6.5/install/kubernetes/

kubectl create \
  -f https://raw.githubusercontent.com/aquasecurity/postee/main/deploy/kubernetes/postee.yaml \
  -f https://raw.githubusercontent.com/aquasecurity/tracee/tracee/deploy/kubernetes/tracee-postee/tracee.yaml

Reply:
After the macro work as expected! I will maintain the git.tag